events/lifestyles/sell_titles_events.txt

```diff
@@ -807,6 +807,10 @@ sell_titles.1002 = {
 	}
 
 	option = {
+		name = {
+			trigger = { like_islam_religion_trigger = yes }
+			text = sell_titles.1002.a.muslim
```

Comment:
` sell_titles.1002.a.muslim:0 "Even Muhammad was a merchant!"`
I'm fairly sure there was no Muhammad in Warcraft universe

Reply:
Commented in 04b7285e02ffb132f63ea560aa13727f7a645782